depthai_nodes/ml/parsers/utils/bbox_format_converters.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
+
+
+def corners_to_rotated_bbox(corners: np.ndarray) -> np.ndarray:
```

Comment:
can we add simple check that the corners order is correct? E.g. check that first corner is actually top-left (smaller x cordinate than top-right etc.)